platform/node/DEVELOPING.md

```diff
@@ -1,19 +1,71 @@
-# Developing the Mapbox GL Native Node.js module
+# Developing the Maplibre GL Native Node.js module
 
-This document explains how to build the [Node.js](https://nodejs.org/) bindings for [Mapbox GL Native](../../README.md) for contributing to the development of the bindings themselves. If you just want to use the module, you can simply install it via `npm`; see [README.md](README.md) for installation and usage instructions.
+This document explains how to build the [Node.js](https://nodejs.org/) bindings for [Maplibre GL Native](../../README.md) for contributing to the development of the bindings themselves. If you just want to use the module, you can simply install it via `npm`; see [README.md](README.md) for installation and usage instructions.
 
 ## Building
 
 To develop these bindings, you’ll need to build them from source. Building requires the prerequisites listed in either
 the [macOS](../macos/INSTALL.md#requirements) or [Linux](../linux/README.md#prerequisites) install documentation, depending
 on the target platform.
 
+First you'll need to install dependencies:
+
+
+#### MacOS
+
+```bash
+brew list cmake || brew install cmake
```

Comment:
Suggestion: remove `brew list <pkg>` from each of following and then use a single `brew install` command so that each package is on a new line:
```bash
brew install \
   cmake \
   ninja \
   ...
```

Reply:
I'd keep it this way to be consistent with existing pipelines: https://github.com/maplibre/maplibre-gl-native/blob/master/.github/workflows/ios-ci.yml#L41-L45

IMO if we were to change them, we should do it all at once, with a consolidated pipeline instead of different per-platform ones.